srcpkgs/mupen64plus/template

```diff
@@ -1,18 +1,19 @@
 # Template file for 'mupen64plus'
 pkgname=mupen64plus
-version=2.5
-revision=13
+version=2.5.9
+revision=1
+archs="i686* x86_64*"
```

Comment:
Please use `nocross=yes` instead of `archs=...`, I can look into fixing it properly later.